neural_speed/models/model_utils/scheduler.h

```diff
@@ -0,0 +1,114 @@
+//  Copyright (c) 2023 Intel Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#ifndef SCHEDULER_H
+#define SCHEDULER_H
+
+#include "models/model_utils/pool.h"
+#include "models/model_utils/model_utils.h"
+
+// iteration-level worker
+class il_worker {
```

Comment:
```il``` seems confusing

Reply:
`il` stands for `iteration-level`. Any suggested changes?

Reply:
```Iter_level_worker_t``` shall be more explicit

Reply:
how about `Iter_level_worker`? since other classes in `neural speed` do not have `_t` suffix.

Reply:
fixed in https://github.com/intel/neural-speed/pull/69/commits/949a463ef3412dcae2971989b678ffd3f62d3271